src/frontends/pytorch/src/op/aminmax.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/reduce_max.hpp"
+#include "openvino/op/reduce_min.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_aminmax(const NodeContext& context) {
```

Comment:
Please add this in the existing file with similar functions: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/pytorch/src/op/min_max.cpp